demo-app/client/generated/controllers/Order.ts

```diff
@@ -45,4 +79,54 @@ export class OrderService {
 
     return this.http.post<object>(`/api/order`, bodyParamsWithoutUndefined, {params: queryParams});
   }
+
+  /**
+   * Patches order
+   * http://example.com/swagger/swagger-ui.html#!/Order/PatchOrder
+   */
+  patchOrder(params: PatchOrderParams): Observable<object> {
+    const pathParams = {
+      orderId: params.orderId,
+    };
+    const bodyParams = {
+      producer: params.producer,
+      model: params.model,
+    };
+    const bodyParamsWithoutUndefined: any = {};
+    Object.entries(bodyParams || {}).forEach(([key, value]) => {
+      if (value !== undefined) bodyParamsWithoutUndefined[key] = value;
+    });
+    return this.http.patch<object>(`/api/order/${pathParams.orderId}`, bodyParamsWithoutUndefined);
+  }
+
+  /**
+   * Put for order
+   * http://example.com/swagger/swagger-ui.html#!/Order/Put-Order
+   */
+  putOrder(params: PutOrderParams): Observable<object> {
+    const pathParams = {
+      orderId: params.orderId,
+    };
+    const bodyParams = {
+      producer: params.producer,
+      model: params.model,
+      customerName: params.customerName,
+    };
+    const bodyParamsWithoutUndefined: any = {};
+    Object.entries(bodyParams || {}).forEach(([key, value]) => {
+      if (value !== undefined) bodyParamsWithoutUndefined[key] = value;
+    });
+    return this.http.put<object>(`/api/order/${pathParams.orderId}`, bodyParamsWithoutUndefined);
+  }
+
+  /**
+   * Delete order
+   * http://example.com/swagger/swagger-ui.html#!/Order/deleteORDER
+   */
+  deleteORDER(params: DeleteORDERParams): Observable<object> {
```

Comment:
Do we really want `ORDER` instead of  `Order`?

Reply:
Yes, I want `findAllByUsingGET` which java generates, and on top of it, although I do not see it crucial, I want that to be backward compatible (the previous *general params* version was not released due to some procedural fault - I am trying to respect the way last release worked).